### PR TITLE
Fix gradle signing inconsistency

### PR DESCRIPTION
**Pull request by atsansone**
Fixes https://github.com/flutter/website/issues/5110

### CODE REVIEW

File: src/content/deployment/android.md [hunk 1]
```diff
@@ -184,74 +184,55 @@ don't check it into public source control.
...
-       keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
-   }
-
-   android {
+When building your app in release mode, configure gradle to use your upload key.
+To configure gradle, edit the `<project>/android/app/build.gradle` file.
+
+1. Add the keystore information from your properties file before the `android` block:
+
+   ```diff
```
Comment (domesticmouse) [inline src/content/deployment/android.md:192]: Why are you indenting this code block?

**Review by domesticmouse — COMMENTED** [review]
I am curious about the intent of MD formatting changes

**Review by sfshaza2 — APPROVED** [review]
lgtm. It's not super critical, but while you're in that file anyway.... Could you replace anchors with gerunds in their names with non-gerunds? This page has got both kinds of headers.

**Comment by atsansone** [comment]
> lgtm. It's not super critical, but while you're in that file anyway.... Could you replace anchors with gerunds in their names with non-gerunds? This page has got both kinds of headers.

Sure! I'll get to it in the morning.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**